tests/__init__.py

```diff
@@ -4,8 +4,8 @@
 SAMPLES_FILES_COUNT: int = 101
 
 # credentials count after scan
-SAMPLES_CRED_COUNT: int = 97
-SAMPLES_CRED_LINE_COUNT: int = 101
+SAMPLES_CRED_COUNT: int = 96
+SAMPLES_CRED_LINE_COUNT: int = 100
```

Comment:
It seems there is affected credential which not detected with this changes.
Could you please check this?

Reply:
That it was fixed. Picatic test sample matched stripe apki key without right barier.
The values are for scan without ml, so i created "GoldenTest"
https://github.com/Samsung/CredSweeper/pull/360

Reply:
I understood. Thank you for explanation.